internal/grid/handlers.go

```diff
@@ -56,6 +56,8 @@ const (
 	HandlerWriteMetadata
 	HandlerCheckParts
 	HandlerRenameData
+	HandlerRenameFile
+	HandlerReadAll
 
 	HandlerServerVerify
```

Comment:
```suggestion
	HandlerServerVerify
	HandlerRenameFile
	HandlerReadAll

```

Keep existing order... Not super important since you are bumping the regular API - but no need to shuffle handler IDs.

Reply:
We haven't made the release yet after this order change so should be okay?